Sources/message/ALMessage.m

```diff
@@ -205,7 +205,7 @@ - (BOOL)isUploadRequire {
 - (BOOL)isHiddenMessage {
     return ((self.contentType == ALMESSAGE_CONTENT_HIDDEN) || [self isVOIPNotificationMessage]
             || [self isPushNotificationMessage] || [self isMessageCategoryHidden]
-            || self.getReplyType== AL_REPLY_BUT_HIDDEN || self.isMsgHidden );
+            || self.getReplyType== AL_REPLY_BUT_HIDDEN || self.isMsgHidden || (self.metadata[KM_ASSIGN_TO] != nil) || (self.metadata[KM_ASSIGN_TEAM] != nil) || (self.metadata[KM_ASSIGN] != nil));
```

Comment:
Can we move these hidden messages to a list or something, then compare vs comparing one by one?

Reply:
removed this here.As we have option to configure it from IOS SDK. Like this 
```
let hiddenMessageMetaDataFlagArray = ["KM_ASSIGN", "KM_STATUS" , "KM_ASSIGN_TO", "KM_ASSIGN_TEAM"]
ALApplozicSettings.hideMessages(withMetadataKeys:hiddenMessageMetaDataFlagArray)
```
Filed PR in IOS SDK -> https://github.com/Kommunicate-io/Kommunicate-iOS-SDK/pull/220